analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExcludeFromCodeCoverageAttributesNeedJustification.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
```

Comment:
This should just work now:
```suggestion
using EFCC = System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute;

...

[EFCC] // Noncompliant
[EFCC(Justification = "justification")] // Compliant
```
Please add the two cases.